crypto/asn1/a_int.c

```diff
@@ -229,12 +229,11 @@ static size_t asn1_put_uint64(unsigned char b[sizeof(uint64_t)], uint64_t r)
 }
 
 /*
- * Absolute value of INT64_MIN: we can't just use -INT64_MIN as it produces
+ * Absolute value of INT64_MIN: we can't just use -INT64_MIN as gcc produces
  * overflow warnings.
  */
 
```

Comment:
This is not exactly reasonable, but all right. For reference, why it's not reasonable? a) Because it was there, and b) it's not the only inconsistency in the file. What I'm trying to say is that [since it's not actually regulated in coding style guide] it shouldn't matter enough to formulate it as *condition* for approval.